blazar.yaml

```diff
@@ -0,0 +1,26 @@
+buildpack:
+  host: git.hubteam.com
+  organization: HubSpotProtected
+  repository: Blazar-Buildpack-Java
+  branch: v2-prebuilt
+
+stepActivation:
+  uploadAndNotify:
+    branches: ['master']
+
+steps:
```

Comment:
Thought process here:
- Compile + test
- Package
- Release 

How do I "release"/upload to Blazar?
I just copied out of the docs the `uploadAndNotify` step here, but not entirely sure what's going on